persistence/postgres-persistence/src/main/java/com/netflix/conductor/postgres/config/outbox/OutboxExecutionDAO.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright 2023 Netflix, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.netflix.conductor.postgres.config.outbox;
+
+import java.sql.Connection;
+import java.util.List;
+
+import javax.sql.DataSource;
+
+import org.springframework.retry.support.RetryTemplate;
+
+import com.netflix.conductor.common.metadata.events.EventExecution;
+import com.netflix.conductor.dao.ExecutionDAO;
+import com.netflix.conductor.model.TaskModel;
+import com.netflix.conductor.model.WorkflowModel;
+import com.netflix.conductor.postgres.dao.PostgresBaseDAO;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Preconditions;
+
+public class OutboxExecutionDAO extends PostgresBaseDAO implements ExecutionDAO {
+
+    private ExecutionDAO delegate;
+
+    public OutboxExecutionDAO(
+            RetryTemplate retryTemplate,
+            ObjectMapper objectMapper,
+            DataSource dataSource) {
+        super(retryTemplate, objectMapper, dataSource);
+        System.out.println("Outbox execution DAO called");
```

Comment:
This is a no no

If you need debug messages, use logging
In the caching proxy somewhere, there is a debug message (or info) that states that caching has been instantiated ... which is a useful piece of information for sure